src/main/java/jsonHandler.java

```diff
@@ -43,7 +44,7 @@ public void saveBuildHistory(BuildHistory db){
     }
 
```

Comment:
Missing documentation. Might come from another branch.

Reply:
This is solved in #44